workflows/agent/model/interview.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pandas as pd
+
+from generative_agents.modeling.agent_helpers import load_llm, load_pinecone
+from generative_agents.modeling.interactions import interview_agent
+from generative_agents.modeling.langchain_agent import GenerativeAgent
+from generative_agents.modeling.memory import create_new_memory_retriever
+from generative_agents.utility import logger, well_known_paths
+from generative_agents.utility.utility import load_params
+
+log = logger.init("conversation")
+
+
+def main():
+    llm = load_llm()
```

Comment:
I suggest passing in the llm type here as well

Reply:
Done! Thanks for inspiring me to do this for all the workflow scripts, have now done it for interview.py, conversations.py, and penpal.py